streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -969,7 +969,8 @@ private boolean close(final long timeoutMs) {
     /**
      * Shutdown this {@code KafkaStreams} by signaling all the threads to stop, and then wait up to the timeout for the
      * threads to join.
-     * A {@code timeout} of 0 means to wait forever.
+     * A {@code timeout} of Duration.ZERO (or any other zero duration) results makes the close operation asynchronous.
```

Comment:
minor comment, but what does `results` indicate here?

Reply:
oops. Sorry about that, revision error.

Reply:
```suggestion
     * A {@code timeout} of Duration.ZERO (or any other zero duration) makes the close operation asynchronous.
```